access/types/forms.py

```diff
@@ -142,6 +142,13 @@ def __init__(self, *args, **kwargs):
 
             g += 1
 
+        if 'lang' in self.exercise:
+            self.add_field(i, {
+                'type': 'hidden',
+                'initial': self.exercise['lang'],
+                'key': 'lang',
```

Comment:
This probably should be something like `__lang` or `__grader_lang`, so there won't be accidental name collisions.